.github/workflows/run-acceptance-tests.yml

```diff
@@ -450,7 +539,7 @@ jobs:
         nodeversion:
           - 14.x
         test-name:
-          - ClusterCs
+          - ClusterGo
```

Comment:
I don't think this is right. I think `test-dotnet:` and `test-go:` nodes each need their own `strategy:` section.

Reply:
yeah I just discovered and fixed it. Learning GHA as I go here.